users/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
+
+from users.apps import UsersConfig
+from users.views import UserCreateAPIView
+
+app_name = UsersConfig.name
+
+urlpatterns = [
+    path("register/", UserCreateAPIView.as_view(), name="register"),
+    path("login/", TokenObtainPairView.as_view(), name="login"),
+    path("token", TokenRefreshView.as_view(), name="token_refresh"),
```

Comment:
Все эндпоинты должны быть однотипны (забыл слеш)
```suggestion
    path("token/", TokenRefreshView.as_view(), name="token_refresh"),
```